docs/plugins/codecs/avro.asciidoc

```diff
@@ -83,6 +83,7 @@ output {
 |=======================================================================
 |Setting |Input type|Required
 | <<plugins-{type}s-{plugin}-ecs_compatibility>> | <<string,string>>|No
+| <<plugins-{type}s-{plugin}-encoding>> | <<string,string>>, one of `["binary", "base64"]`|No
```

Comment:
Watchlist: Correction made in source. Recent enhancements to docs tooling to allow this content to stand rather than getting stomped in subsequent docgens.